frontend/src/sass/pages/_meet-the-team.scss

```diff
@@ -16,11 +16,28 @@ $team-individual-img-height: 150px;
 
 .team {
 	
+	display: flex;
+	
+	&__first-column {
```

Comment:
Hi Lili, yes I can use the responsive grid but they won't tile up nicely, they will be placed in rows.